### DISCUSSION TITLE
Automatic night mode in tunnel

### DISCUSSION THREAD

**Discussion by toas-koas** [organicmaps/organicmaps#4393] [Ideas]
It would be good if the app automatically changes to the night mode as soon as I enter a tunnel.
If I leave the tunnel, the day mode should be activated again.
This could be automated wherever a tunnel is entered in OpenStreetMap.

**Comment by pm4rcin** [comment]
Magic Earth has that feature. Their implementation is based on light sensor so it automatically switches since it's generally darker in tunnels.

**Reply by toas-koas** [reply]
In ME it's based on osm data and not the light sensor.

**Reply by biodranik** [reply]
OSM data may be wrong. A sensor is more reliable. There is no reason to use dark mode under a bright light.
